From b0fa91190e498f003bebd45b81b41f7993526d51 Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@apache.org>
Date: Thu, 31 Mar 2016 13:09:09 -0700
Subject: [PATCH 1421/2748] YARN-4639. Remove dead code in
 TestDelegationTokenRenewer added in YARN-3055
 (templedf via rkanter)

(cherry picked from commit 7a021471c376ce846090fbd1a315266bada048d4)
(cherry picked from commit 633f612d6773442a9fa64ca9843c2b0e676c309a)

Change-Id: I44a07fb7ae3076d6a0f00e0818cd4027f2dcc444
---
 .../security/TestDelegationTokenRenewer.java       |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/security/TestDelegationTokenRenewer.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/security/TestDelegationTokenRenewer.java
index 9481448..370e016 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/security/TestDelegationTokenRenewer.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/security/TestDelegationTokenRenewer.java
@@ -1077,9 +1077,6 @@ public void testCancelWithMultipleAppSubmissions() throws Exception {
         new MockNM("127.0.0.1:1234", 15120, rm.getResourceTrackerService());
     nm1.registerNode();
 
-    //MyFS fs = (MyFS)FileSystem.get(conf);
-    //MyToken token1 = fs.getDelegationToken("user123");
-
     // create Token1:
     Text userText1 = new Text("user");
     DelegationTokenIdentifier dtId1 =
-- 
1.7.9.5

